ci/scripts/lint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -eux
+
+pushd sixteens
+  npm install --unsafe-perm
```

Comment:
do you really need to spend time installing if we're just exiting?

Reply:
just talked about that when updating the pipeline, probably not (it was primarily a copy and paste from another job)